company_linkage/airtable_configs/parat_validate/parent_validate.json

```diff
@@ -5,8 +5,6 @@
   "merge_query": "parent_merge",
   "production_table": "parent_validate",
   "column_map": {
-    "CSET_id": "CSET_id",
-    "name": "name",
     "parent_acquisition": "parent_acquisition",
```

Comment:
It looks like based on your changes I'm not supposed to map columns that have the same names -- should I delete these three rows as well then?

I assume I should keep the final one ("annotator_comment": "comment" since I'm actually updating the name here, but the other three aren't...

Reply:
Yeah - you only need to populate the column map if you're renaming or excluding the column; looks like I just missed deleting those rows so good catch, you can delete them too if you want